torchx/mix.exs

```diff
@@ -21,11 +22,8 @@ defmodule Torchx.MixProject do
       aliases: aliases(),
       make_env: %{
         "LIBTORCH_DIR" => System.get_env("LIBTORCH_DIR", @libtorch_cache),
-        "BUILD_EMBEDDED" =>
-          System.get_env(
-            "BUILD_EMBEDDED",
-            "#{Keyword.get(Mix.Project.config(), :build_embedded)}"
-          )
+        "LIBTORCH_BASE" => @libtorch_base,
```

Comment:
Doesn't this become "wrong" if I set LIBTORCH_DIR, for instance, to a CUDA version?

I mean, it would work, but it wouldn't have the correct name

Reply:
You are right. If the env var is set, should we set base to something like "custom"?

Reply:
Please check again!